docs/source/guide/helm_values.md

```diff
@@ -216,21 +223,24 @@ Parameters specific to the `rqworkers` service of your Label Studio Enterprise d
 | `rqworker.containerSecurityContext.runAsNonRoot`             | string   | true     | Avoid running as root User                                                                  |
 | `rqworker.containerSecurityContext.allowPrivilegeEscalation` | string   | false    | Controls whether a process can gain more privileges than its parent process                 |
 | `rqworker.topologySpreadConstraints`                         | list     | [ ]      | Topology Spread Constraints for pod assignment                                              |
-| `rqworker.dnsPolicy`                                         | string   | ""       | Pod DNS policy                                                                              |
+| `rqworker.dnsPolicy`                                         | string   | [ ]      | Pod DNS policy                                                                              |
```

Comment:
if it's string, why [] are there?

Reply:
@makseq should this be true or false on Line #226

Reply:
it should be `""`